platforms/r3-corda-ent/configuration/roles/helm_component/templates/notary.tpl

```diff
@@ -1,10 +1,6 @@
-apiVersion: flux.weave.works/v1beta1
```

Comment:
This is incorrect merge : change this file to what we have in feature/helm3 branch.